compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -806,6 +806,8 @@ static LogicalResult setMatmulPadRootConfig(
   tileSizes.emplace_back(flowTileSizes.begin(), flowTileSizes.end());
   tileSizes.push_back(parallelTileSizes);
   tileSizes.push_back(reductionTileSizes);
+  // No need for tiling inner parallel dims.
+  tileSizes.emplace_back(parallelTileSizes.size(), 0);
```

Comment:
nit: could we replace the `XYZTileSizes.size()` with `numTilingDims` or similar? That would make it easy to read and more explicit about any assumptions needed to understand this code.